<launch>
  <node pkg="prophesee_ros_stereo_driver" type="prophesee_ros_stereo_publisher" name="prophesee_ros_stereo_publisher" respawn="false" output="screen" >
    <param name="camera_name" value="camera" />
    <param name="left_camera_id" type="string" value="00002400" />
    <param name="right_camera_id" type="string" value="00002402" />
    <param name="master_left" value="true" />
    <param name="publish_cd" value="true" />

    <!-- Path to the file with the camera settings -->
    <param name="bias_file" value="" />
  </node>
</launch>
